src/figma-default-layers.ts

```diff
@@ -0,0 +1,604 @@
+    strokeRightWeight: 1,
+    type: "RECTANGLE"
+  },
+  LINE: {
+    id: "_",
+    name: "Line",
+    removed: false,
+    visible: true,
+    locked: false,
+    componentPropertyReferences: null,
+    opacity: 1,
+    blendMode: "PASS_THROUGH",
+    isMask: false,
+    effects: [],
+    effectStyleId: "",
+    fills: [],
+    fillStyleId: "",
+    strokes: [
+      {
+        type: "SOLID",
+        visible: true,
+        opacity: 1,
+        blendMode: "NORMAL",
+        color: {
+          r: 0,
+          g: 0,
+          b: 0
+        }
+      }
+    ],
+    strokeStyleId: "",
+    strokeWeight: 1,
+    strokeAlign: "CENTER",
+    strokeJoin: "MITER",
+    dashPattern: [],
+    strokeCap: "NONE",
+    strokeMiterLimit: 4,
+    fillGeometry: [],
+    strokeGeometry: [
+      {
+        windingRule: "NONZERO",
+        data: "M0 0L100 0L100 -1L0 -1L0 0Z"
+      }
+    ],
+    relativeTransform: [
+      [1, 0, 0],
+      [0, 1, 0]
+    ],
+    x: 0,
+    y: 0,
+    width: 100,
+    height: 0,
+    rotation: 0,
+    layoutAlign: "INHERIT",
+    constrainProportions: false,
+    layoutGrow: 0,
+    layoutPositioning: "AUTO",
+    exportSettings: [],
+    constraints: {
+      horizontal: "MIN",
+      vertical: "MIN"
+    },
+    reactions: [],
+    type: "LINE"
+  },
+  ELLIPSE: {
+    id: "_",
+    name: "Ellipse",
+    removed: false,
+    visible: true,
+    locked: false,
+    componentPropertyReferences: null,
+    opacity: 1,
+    blendMode: "PASS_THROUGH",
+    isMask: false,
+    effects: [],
+    effectStyleId: "",
+    fills: [
+      {
+        type: "SOLID",
+        visible: true,
+        opacity: 1,
+        blendMode: "NORMAL",
+        color: {
+          r: 0.8509804010391235,
+          g: 0.8509804010391235,
+          b: 0.8509804010391235
+        }
+      }
+    ],
+    fillStyleId: "",
+    strokes: [],
+    strokeStyleId: "",
+    strokeWeight: 1,
+    strokeAlign: "INSIDE",
+    strokeJoin: "MITER",
+    dashPattern: [],
+    strokeCap: "NONE",
+    strokeMiterLimit: 4,
+    fillGeometry: [
+      {
+        windingRule: "NONZERO",
+        data: "M100 50C100 77.6142 77.6142 100 50 100C22.3858 100 0 77.6142 0 50C0 22.3858 22.3858 0 50 0C77.6142 0 100 22.3858 100 50Z"
+      }
+    ],
+    strokeGeometry: [],
+    relativeTransform: [
+      [1, 0, 0],
+      [0, 1, 0]
+    ],
+    x: 0,
+    y: 0,
+    width: 100,
+    height: 100,
+    rotation: 0,
+    layoutAlign: "INHERIT",
+    constrainProportions: false,
+    layoutGrow: 0,
+    layoutPositioning: "AUTO",
+    exportSettings: [],
+    constraints: {
+      horizontal: "MIN",
+      vertical: "MIN"
+    },
+    cornerRadius: 0,
+    cornerSmoothing: 0,
+    arcData: {
+      startingAngle: 0,
+      endingAngle: 6.2831854820251465,
+      innerRadius: 0
+    },
+    reactions: [],
+    type: "ELLIPSE"
+  },
+  POLYGON: {
+    id: "_",
+    name: "Polygon",
+    removed: false,
+    visible: true,
+    locked: false,
+    componentPropertyReferences: null,
```

Comment:
hmm, why is this here?

Reply:
Extra info that might prove useful in the future: https://www.figma.com/plugin-docs/api/node-properties/#componentpropertyreferences. We won't need it depending on how we'll handle components props but left it in in case we do.It comes by default with Figma layers and shouldn't be actively harmful

Reply:
Interesting. Wondering if we should keep that modeled on the component parent or the sublayer. fine for now.